Fix `abs()` warning in `vectorio/Circle.c`

Tagging @kvc0 and maybe @FoamyGuy:

Compilation generates this mild warning:
```
../../shared-module/vectorio/Circle.c: In function 'common_hal_vectorio_circle_get_pixel':
../../shared-module/vectorio/Circle.c:26:22: warning: taking the absolute value of unsigned type 'uint16_t' {aka 'short unsigned int'} has no effect [-Wabsolute-value]
   26 |     int16_t radius = abs(self->radius);
      |                      ^~~
```
The `radius` struct element is already unsigned:
https://github.com/adafruit/circuitpython/blob/5014ddb43c28f7a5b7ac38ef9bf65c65d6006597/shared-module/vectorio/Circle.h#L12
I am not quite sure what is going on here with the `x` and `y` here, because they are going to be `>0` after lines 27 and 28. Maybe the original arguments should have been preserved?
https://github.com/adafruit/circuitpython/blob/5014ddb43c28f7a5b7ac38ef9bf65c65d6006597/shared-module/vectorio/Circle.c#L24-L40

--- a/shared-module/vectorio/Circle.c
+++ b/shared-module/vectorio/Circle.c
@@ -23,7 +23,7 @@ void common_hal_vectorio_circle_set_on_dirty(vectorio_circle_t *self, vectorio_e
 
 uint32_t common_hal_vectorio_circle_get_pixel(void *obj, int16_t x, int16_t y) {
     vectorio_circle_t *self = obj;
-    int16_t radius = abs(self->radius);
+    int16_t radius = self->radius;
     x = abs(x);
     y = abs(y);
     if (x + y <= radius) {

